sdk/spring/azure-spring-boot-samples/README.md

```diff
@@ -31,12 +31,19 @@ It means cloning the master branch, all the code, dependencies, and readme files
 
 2. Traverse to the root directory
 
-3. Build the whole product by executing the following command which may take several minutes
+3. Build modules
+
+   1. Build all modules under the `spring` module by executing the following command which may take several minutes.
```

Comment:
I suggest we remove this section since it will fail when the dependencies are not built on the local machine, such as EH sdk.